WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -75,11 +75,15 @@ def users_search
     search(User)
   end
 
+  def regulations_search
+    search(Regulation)
```

Comment:
Have you checked that it works fine? Other models has defined a method `serializable_hash` and I guess it's meant to provide the json.